acra/configuring-maintaining/general-configuration/acra_server.md

```diff
@@ -0,0 +1,419 @@
+  Can't be used with `--tls_client_auth=0` or `--tls_auth=0`.
+  Default is `false`.
+
+* `--tls_database_auth=<mode>`
+
+  Set authentication mode that will be used for TLS connection with database.
+  Possible values are the same as for `--tls_auth`.
+  Overrides the `--tls_auth` setting.
+  Default is `-1` which means "take value of `--tls_auth`".
+
+* `--tls_database_key=<filename>`
+
+  Path to private key that will be used for TLS handshake with database.
+  Should correspond to the certificate configured with `--tls_database_cert`.
+  Empty by default.
+
+* `--tls_database_cert=<filename>`
+
+  Path to client TLS certificate shown to database during TLS handshake (overrides `--tls_cert`).
+  Empty by default.
+
+* `--tls_database_ca=<filename>`
+
+  Path to additional CA certificate for database certificate validation
+  (setup if database certificate CA is different from AcraConnector certificate CA).
+  Empty by default.
+
+* `--tls_database_sni=<SNI>`
+
+  Expected Server Name (SNI) from database.
+  Empty by default which means "don't check the SNI sent by database".
+
+* `--tls_db_sni=<SNI>`
+
+  Expected Server Name (SNI) from database (deprecated, use `--tls_database_sni` instead).
+
+* `--tls_identifier_extractor_type=<type>`
+
+  Decide which field of TLS certificate to use as ClientID.
+
+  * `distinguished_name` — (default) certificate Distinguished Name (DN)
+  * `serial_number` — certificate serial number
+
+### Jaeger
+
+* `--jaeger_agent_endpoint=<addr>`
+
+  Jaeger agent endpoint that will be used to export trace data.
+  Example: `localhost:6831`.
+  Default is empty.
+
+* `--jaeger_basic_auth_password=<password>`
+
+  Password used for basic auth (optional) to jaeger.
+
+* `--jaeger_basic_auth_username=<username>`
+
+  Username used for basic auth (optional) to jaeger.
+
+* `--jaeger_collector_endpoint=<url>`
+
+  Jaeger endpoint that will be used to export trace data.
+  Example: `http://localhost:14268/api/traces`.
+  Default is empty.
+
+* `--tracing_jaeger_enable={true|false}`
+
+  Export trace data to jaeger.
+  Default is `false`.
+
+### Logging
+
+* `--log_to_console={true|false}`
+
+  Log to stderr.
+  Default is `true`.
+
+* `--log_to_file=<filename>`
+
+  Log to file if non-empty value was passed.
+  Default is empty.
+
+* `--logging_format={plaintext|json|CEF}`
+
+  Logging format.
+
+  * `plaintext` — (default) pretty human readable key/value format  
+    `time="2021-07-12T14:02:12+03:00" level=info msg="Starting service acra-translator [pid=475995]" version=0.85.0`
+
+  * `json` — one JSON object per line, easy to parse by most log collectors  
+    `{"level":"info","msg":"Starting service acra-translator [pid=476077]","product":"acra-translator","timestamp":"2021-07-12T14:02:50+03:00","unixTime":"1626087770.004","version":"0.85.0"}`
+
+  * `CEF` — Common Event Format  
+    `CEF:0|cossacklabs|acra-translator|0.85.0|100|Starting service acra-translator [pid\=476133]|1|unixTime=1626087782.510`
```

Comment:
Fenced blocks would render more nicely:

```markdown
  * `plaintext` — (default) pretty human readable key/value format

    ```
    time="2021-07-12T14:02:12+03:00" level=info msg="Starting service acra-translator [pid=475995]" version=0.85.0
    ```
```

Also, it would be nice to stop this trailing whitespace thing. A lot of editors would strip them by default and won't show them. Use `<br>` if you need a line break.